docs/admin/install.rst

```diff
@@ -0,0 +1,243 @@
+
+- Copy the Journalist Interface configuration file to `dom0`. If your SecureDrop instance uses v3 onion services, use the following command:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault "cat /run/media/user/TailsData/Persistent/securedrop/install_files/ansible-base/app-journalist.auth_private" > /tmp/journalist.txt
+
+  If your instance uses legacy v2 onion services, use the following command instead:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault "cat /run/media/user/TailsData/Persistent/securedrop/install_files/ansible-base/app-journalist.aths" > /tmp/journalist.txt
+
+- Verify that the `/tmp/journalist` file on ``dom0`` contains valid configuration information using the command ``cat /tmp/journalist.txt`` in the administration terminal.
+
+- In the ``vault`` file manager, select **+ Other Locations** and eject the TailsData volume, then disconnect the Admin Workstation USB. Shut down the ``vault`` VM using the Qube widget in the upper right panel.
+
+
+Download and install SecureDrop Workstation
+-------------------------------------------
+
+With the key and configuration available in ``dom0``, you're ready to set up SecureDrop Workstation:
+
+- First, re-enable the network connection using the network manager widget.
+
+- Next, start a terminal in the network-attached ``work`` VM, via **Q > Domain:work > work: Terminal**.
+
+- In the ``work`` terminal, run the following commands to download and add the SecureDrop signing key, which is needed to verify the SecureDrop Workstation package:
+
+  .. code-block:: sh
+
+    gpg --keyserver hkps://keys.openpgp.org --recv-key "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
+    gpg --armor --export 22245C81E3BAEB4138B36061310F561200F4AD77 | sudo tee /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation
+
+- In the ``work`` terminal, create a file ``/etc/yum.repos.d/securedrop-temp.repo`` with the following contents:
+
+  .. code-block:: none
+
+    [securedrop-workstation-temporary]
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation
+    enabled=1
+    baseurl=https://yum.securedrop.org/workstation/dom0/f25
+    name=SecureDrop Workstation Qubes initial install bootstrap
+
+- Download the SecureDrop Workstation config package with the command:
+
+  .. code-block:: sh
+
+    dnf download securedrop-workstation-dom0-config
+
+- In the ``dom0`` administration terminal, run the following command to transfer the RPM package to dom0 and install SecureDrop Workstation:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io work 'cat /home/user/securedrop-workstation-dom0-config-<versionNumber>-1.fc25.noarch.rpm' > securedrop-workstation.rpm
+    sudo dnf install securedrop-workstation.rpm
+
+  where ``<versionNumber>`` is the current SecureDrop Workstation release version. When prompted, press **Y** to install the package.
```

Comment:
Since they've just downloaded it, we can instruct them to note down the version number after the download.

Reply:
done.